x-pack/plugins/code/public/components/admin_page/project_tab.tsx

```diff
@@ -205,7 +206,7 @@ class CodeProjectTab extends React.PureComponent<Props, State> {
         project={repo}
         showStatus={true}
         status={status[repo.uri]}
-        enableManagement={isAdmin}
+        enableManagement={uiCapabilities.code.admin as boolean} // TODO check if this is the right use
```

Comment:
Is this `TODO` for me, or someone on the code team? `uiCapabilities.code.admin` is the correct replacement for `isAdmin`.

Reply:
done